app/models/comment.rb

```diff
@@ -0,0 +1,16 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def update_comment_counter
+    post.increment!(:comments_counter)
+  end
+end
+# User.update_post_counter
+# Class.Thenameofthemethod  6
+# Comment.create(post_id: first_post.id, author_id: second_user.id, text: 'Hi Tom!' )
+# Comment.create(post_id: second_post.id, author_id: second_user.id, text: 'Hi Lilly!' )
+# Comment.create(post_id: first_post.id, author_id: second_user.id, text: 'Hi Tom!' )
+# Comment.create(post_id: third_post.id, author_id: second_user.id, text: 'Hi Vanina!' )
+# Comment.create(post_id: first_post.id, author_id: second_user.id, text: 'Hi Lari!' )
+# Comment.create(post_id: second_post.id, author_id: second_user.id, text: 'Hi Tom!' )
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1:
- [ OPTIONAL ] Instead of leaving commented pieces of code in your codebase, for the creation of comments, users, and posts to populate the database, you can consider making use of the `seed.rb` file. All you need to do is to include this code for creating users, comments, posts etc. in it uncommented after which you can run `rails db:seed` to create them all and save them to the database :+1: 


- You can read more about it [here](https://marketsplash.com/tutorials/ruby-on-rails/ruby-on-rails-how-to-work-with-seed-file/#:~:text=Once%20you%20have%20navigated%20to,with%20the%20data%20you%20defined.&text=This%20command%20will%20execute%20the,with%20the%20data%20you%20defined.) :100: